specification/dsdl/serializable_types.tex

```diff
@@ -195,20 +225,22 @@ \subsection{Array types}
 \end{itemize}
 If neither of the above prefixes are provided, the resultant definition is that of a fixed-length array.
 
+The alignment of an array equals the alignment of its element type\footnote{
+    E.g., the alignment of \texttt{uint5[<=3]} or \texttt{int64[3]} is 1 bit (that is, no alignment).
+}.
+
 \subsection{Composite types}\label{sec:dsdl_composite_types}
 
 \subsubsection{Kinds}
 
-There are two kinds of composite data type definitions: message types and service types.
+There are two kinds of composite type definitions: message types and service types.
 All versions of a data type shall be of the same kind\footnote{%
     For example, if a data type version 0.1 is of a message kind, all later versions of it shall be messages, too.
 }.
 
-A message data type defines one data schema.
-
-A service data type contains two data schema definitions:
+A service type defines two unutterable inferior data types:
```

Comment:
They are "unutterable inferior types". An unutterable type (also known as [Voldemort type](https://wiki.dlang.org/Voldemort_types)) is a type that cannot be named. I think this is a fairly common slang term, do you think it is unacceptable here?

Reply:
Ah, no. I somehow missed this slang although I have heard of Voldemort types. I think putting those two words together makes for something that sound like a joke instead of a specification. If we can find a better term then "inferior" I think we could keep unutterable. Let's not have people refer to their carefully crafted data types as "a bunch of inferior crap so bad people can't even bring themselves to talk about it." :)